scripts/install.ps1

```
@@ -0,0 +1,60 @@
+$ZipPath = "$env:TEMP\mdmend.zip"
+
+Write-Host "Downloading mdmend v$Version for $Arch..."
+try {
+    Invoke-WebRequest -Uri $Url -OutFile $ZipPath
+} catch {
+    Write-Error "Failed to download binary from $Url. Please ensure version v$Version exists for $Arch."
+    exit 1
+}
+
+# Extract
+Write-Host "Installing to $InstallDir..."
```

PSScriptAnalyzer (1.24.0)

[warning] File 'install.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)